Documentation/Reference.md

```diff
@@ -61,51 +61,117 @@ Where `foo` is an executable and `bar.a` a static library.
 
 ## Package Manifest File Format Reference
 
-Instructions for how to build a package are provided by a manifest file, called `Package.swift`. You can customize this file to declare build targets or dependencies, include or exclude source files, and specify build configurations for the module or individual files.
+Instructions for how to build a package are provided by the `Package.swift` manifest file. `Package.swift` is a Swift file that declaratively configures a Package using types defined in the `PackageDescription` module. It currently has following properties:
 
```

Comment:
It might be good to have an overview here explaining that this is a swift file defining a single Package object, and the package is configured via the APIs used to form that object.